### PR TITLE
Fix refactorability for "owner"

### PR DESCRIPTION
**Pull request by mvorisek**
Important for quick navigation and refactoring.

### CODE REVIEW

File: src/Field_SQL.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 /**
  * Class description?
+ *
+ * @property Join\SQL $join
```
Comment (mvorisek) [inline src/Field_SQL.php:13]: Is this assumtion right, or can be here non-SQL Join?

File: src/Join/SQL.php [hunk 2]
```diff
@@ -9,6 +9,9 @@
 
 /**
  * Join\SQL class.
+ *
+ * @property \atk4\data\Persistence\SQL $persistence
```
Comment (mvorisek) [inline src/Join/SQL.php:13]: Is this assumtion right, or can be here non-SQL Persistence?

File: src/Join/SQL.php [hunk 3]
```diff
@@ -9,6 +9,9 @@
...
+ * @property SQL $join
```
Comment (mvorisek) [inline src/Join/SQL.php:14]: Is this assumtion right, or can be here non-SQL Join?

File: src/Model.php [hunk 4]
```diff
@@ -21,6 +21,8 @@
 
 /**
  * Data model class.
+ *
+ * @property Field[]|Reference[] $elements
```
Comment (mvorisek) [inline src/Model.php:25]: Is this assumtion right?

Comment (DarkSide666) [reply src/Model.php:25]: Currently yes, but in future I think more element types could go there. Maybe even scopes/conditions, joins etc. Not sure, but good for now.

**Pull request merged**